sensor/common/signal/signal_service.go

```diff
@@ -139,40 +81,32 @@ func (s *serviceImpl) receiveMessages(stream sensorAPI.SignalService_PushSignals
 			return err
 		}
 
-		// Ignore the collector register request
-		if signalStreamMsg.GetSignal() == nil {
-			log.Error("Empty signalStreamMsg")
-			continue
+		signal, err := unwrapSignal(signalStreamMsg)
+		if err != nil {
+			metrics.IncrementTotalProcessesDroppedCounter()
```

Comment:
🤔 I'm not sure about this metric. Normally these `SomethingDroppedCounter` are used to record when we drop a message not because it's not valid but because the buffers are full.

I think this should be used when we are pushing messages to the queue:
```golang
select {
case s.queue <- signal:
	metrics.IncrementTotalProcessesReceivedCounter()
default:
	metrics.IncrementTotalProcessesDroppedCounter()
}
```